src/main/resources/lang/installer.properties

```
@@ -30,5 +30,7 @@ prompt.server.jar.valid=Valid {0} server jar found
 prompt.server.jar.invalid=No valid {0} server jar found
 prompt.server.generate=Generate
 prompt.server.overwrite=Are you sure you want to override the existing launch scripts?
+prompt.install.successful.title=Successfully Installed
+prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Most mods will also require you to install <a href="{2}">Fabric API</a>.
```

I think this message could be improved. Something to clarify that Fabric API is just a mod, and doesn't have an installer or anything. 

Perhaps:
```suggestion
prompt.install.successful=Fabric Loader {0} for {1} has been successfully installed.<br>Many mods also depend on the <a href="{2}">Fabric API</a> mod which you can download and put in the mods folder in your instance.
```

---

Yes, it should mention to put it into the mods folder. Something along the lines of what @Prospector said put possibly removing the "in your instance."

---

Changed to "Many mods also require you to download [Fabric API](https://www.curseforge.com/minecraft/mc-mods/fabric-api/) mod into the mods folder."

---

hmm, should have `the` in front of Fabric API mod but also I'm not sure the phrasing on "download ... into the mods folder" is very great. I'd just use what I put above.

---

`Many mods also depend on the Fabric API which you can download and put in the mods folder` is really long tbh.

how about this?
`Many mods also require you to put the Fabric API mod into the mods folder.`

---

I would go with `Many mods also require you to put Fabric API into the mods folder.`

---

I don't believe the installer creates a mods folder, it only appears on first launch. You can expect that mentioning a nonexisting mods folder would be confusing to some people.

If we are going to mention the mods folder I would suggest having the installer create the mods folder, and adding a button in the installer to open it. That's a bit out of scope for this PR but I think it would be a useful improvement in the future.

---

Humm, that is a good point. The installer could possibly create the mods folder when creating the profile, but then this would only work when the profile is creted (I assume most people leave this enabled).

I dont think this is a blocker for merging this PR however

---

tbf people should know what a mods folder is, that's the minimum amount of knowledge to get the game modded.

---

Updated the text to `Many mods also require you to put Fabric API into the mods folder.`

---

One more alternative, just to throw it out there:
`Most mods also require the Fabric API mod to be installed.`
Why mention the mods folder and make it longer?